src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
```suggestion
    private static final Random random = new Random();
    
    public Colors getRandomColor() {
        int index = random.nextInt(Colors.values().length);
```
lest make random global
this way only 1 instance of Random will be created, instead of each time you invoke the method

this method should not be static